parsers/rhea/rhea_test.go

```diff
@@ -0,0 +1,58 @@
+package rhea
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+	"os"
+	"testing"
+)
+
+var db *sql.DB
+var rhea Rhea
+
+func TestMain(m *testing.M) {
+	// Read the Compressed Rhea XML to bytes
+	rheaBytes, err := ReadRhea("data/rhea.rdf.gz")
+	if err != nil {
+		log.Fatalf("Failed to read rhea")
+	}
+
+	// Parse the Rhea bytes into the rhea struct
+	rhea, err = ParseRhea(rheaBytes)
+	if err != nil {
+		log.Fatalf("Failed to parse rhea")
+	}
+
+	// Start running tests
+	code := m.Run()
+	os.Exit(code)
+}
+
+func ExampleExportRhea() {
+	// Convert rhea to JSON
+	rheaJson, _ := rhea.Export()
+
+	fmt.Println(string(rheaJson)[:100])
```

Comment:
Since this data structure is so confusing, having a few (hopefully very short/minimal) before and after parsings explicitly printed out would be very helpful.
```suggestion
	fmt.Println(string(rheaJson.smallExample))
```

Reply:
Do you have any suggestions for this? The rheaJson is literally just some bytes, so I was having a hard time figuring out a good way to make "good" examples.